packages/models/src/contract/contract.ts

```diff
@@ -36,7 +35,7 @@ export class Contract<
         status: Status.ok,
         message: {
           pattern: payload.pattern,
-          value: res.body.json(),
+          value: res.json(),
```

Comment:
`res.json()` returns a promise that should be resolved before being returned.

Ref:
- json: https://developer.mozilla.org/en-US/docs/Web/API/Response/json